contracts/core/exchange-wrappers/KyberNetworkWrapper.sol

```diff
@@ -171,6 +174,20 @@ contract KyberNetworkWrapper {
         );
     }
 
+    /**
+     * Set the CoreIsssuanceModule so that only it can call exchange.
+     *
+     * @param  _newIssuanceOrderModule          Address of deployed issuanceOrderModule contract
+     */    
+    function setIssuanceOrderModule(
```

Comment:
I think as little redeployment of contracts as possible is best. Not necessarily because its cheaper but because we should try to make the system as static as possible, people in theory will be watching this contracts quite a bit and moving them around with a redeploy feels less scalable.